packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -254,6 +254,17 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
         return _vaults[validator].asset.validatorKroBonded;
     }
 
+    /**
+     * @notice Returns the total amount of validator balance excluding the bond amount.
+     *
+     * @param validator Address of the validator.
+     *
+     * @return The total amount of validator balance excluding the bond amount.
+     */
+    function totalValidatorBalance(address validator) external view returns (uint128) {
```

Comment:
How about the name like `totalValidatorNotBonded` or `totalValidatorRemainder`?

Reply:
I tried to give it a similar meaning as in V1 (the `balanceOf` function). Won't those names be not much intuitive?

Reply:
`totalValidatorKro` and `totalValidatorKroBonded` do already exist in V2, so it might be confusing imo..

Reply:
I agree. Then I'll change it to totalValidatorKroNotBonded`.